torchmultimodal/models/video_vqvae.py

```diff
@@ -213,3 +278,20 @@ def __init__(self, hidden_dim: int = 240, n_head: int = 2) -> None:
 
     def forward(self, x: Tensor) -> Tensor:
         return x + self.block(x)
+
+
+def _preprocess_int_conv_params(
+    channel_dims: Tuple[int, ...],
+    kernel_sizes: int,
+    strides: int,
+) -> Tuple:
```

Comment:
nit: you could make a namedtuple for the return type here to make the typing more explicit